app/src/main/java/co/icanteach/projectx/common/di/module/ViewModelModule.kt

```diff
@@ -7,9 +7,12 @@ import co.icanteach.projectx.common.di.key.ViewModelKey
 import co.icanteach.projectx.ui.populartvshows.PopularTVShowsViewModel
 import dagger.Binds
 import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ApplicationComponent
 import dagger.multibindings.IntoMap
 
 @Module
+@InstallIn(ApplicationComponent::class)
 interface ViewModelModule {
```

Comment:
I would be nice if ViewModel is also migrated to Hilt?

https://developer.android.com/training/dependency-injection/hilt-jetpack